flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/catalog/CatalogTableITCase.scala

```diff
@@ -877,6 +877,397 @@ class CatalogTableITCase(isStreamingMode: Boolean) extends AbstractTestBase {
+        |)
+      """.stripMargin
+
+    val view1DDL =
+      """
+        |CREATE VIEW T2(d, e, f) AS SELECT a, b, c FROM T1
+      """.stripMargin
+
+    val view2DDL =
+      """
+        |CREATE VIEW T3(x, y, z) AS SELECT a, b, c FROM T1
+      """.stripMargin
+
+    tableEnv.sqlUpdate(sourceDDL)
+    tableEnv.sqlUpdate(view1DDL)
+    tableEnv.sqlUpdate(view2DDL)
+
+    assert(tableEnv.listTables().sameElements(Array[String]("T1", "T2", "T3")))
+
+    tableEnv.sqlUpdate("DROP VIEW T2")
+    assert(tableEnv.listTables().sameElements(Array[String]("T1", "T3")))
+
+    tableEnv.sqlUpdate("DROP VIEW default_database.T3")
+    assert(tableEnv.listTables().sameElements(Array[String]("T1")))
+  }
+
+  @Test
+  def testDropViewIfExistsTwice(): Unit = {
+    val sourceDDL =
+      """
+        |CREATE TABLE T1(
+        |  a int,
+        |  b varchar,
+        |  c int
+        |) with (
+        |  'connector' = 'COLLECTION'
+        |)
+      """.stripMargin
+
+    val viewDDL =
+      """
+        |CREATE VIEW T2(d, e, f) AS SELECT a, b, c FROM T1
+      """.stripMargin
+
+
+    tableEnv.sqlUpdate(sourceDDL)
+    tableEnv.sqlUpdate(viewDDL)
+
+    assert(tableEnv.listTables().sameElements(Array[String]("T1", "T2")))
+
+    tableEnv.sqlUpdate("DROP VIEW IF EXISTS default_catalog.default_database.T2")
+    assert(tableEnv.listTables().sameElements(Array[String]("T1")))
+
+    tableEnv.sqlUpdate("DROP VIEW IF EXISTS default_catalog.default_database.T2")
+    assert(tableEnv.listTables().sameElements(Array[String]("T1")))
+  }
+
+  @Test(expected = classOf[ValidationException])
+  def testDropViewTwice(): Unit = {
+    val sourceDDL =
+      """
+        |CREATE TABLE T1(
+        |  a int,
+        |  b varchar,
+        |  c int
+        |) with (
+        |  'connector' = 'COLLECTION'
+        |)
+      """.stripMargin
+
+    val viewDDL =
+      """
+        |CREATE VIEW T2(d, e, f) AS SELECT a, b, c FROM T1
+      """.stripMargin
+
+
+    tableEnv.sqlUpdate(sourceDDL)
+    tableEnv.sqlUpdate(viewDDL)
+
+    assert(tableEnv.listTables().sameElements(Array[String]("T1", "T2")))
+
+    tableEnv.sqlUpdate("DROP VIEW default_catalog.default_database.T2")
+    assert(tableEnv.listTables().sameElements(Array[String]("T1")))
+
+    tableEnv.sqlUpdate("DROP VIEW default_catalog.default_database.T2")
+  }
+
+  @Test(expected = classOf[ValidationException])
+  def testDropViewWithInvalidPath(): Unit = {
+    val sourceDDL =
+      """
+        |CREATE TABLE T1(
+        |  a int,
+        |  b varchar,
+        |  c int
+        |) with (
+        |  'connector' = 'COLLECTION'
+        |)
+      """.stripMargin
+
+    val viewDDL =
+      """
+        |CREATE VIEW T2(d, e, f) AS SELECT a, b, c FROM T1
+      """.stripMargin
+    tableEnv.sqlUpdate(sourceDDL)
+    tableEnv.sqlUpdate(viewDDL)
+    assert(tableEnv.listTables().sameElements(Array[String]("T1", "T2")))
+    tableEnv.sqlUpdate("DROP VIEW default_catalog1.default_database1.T2")
```

Comment:
both `default_catalog1 ` and `default_database1 ` does not exist ?

add some comments before the failed case

Reply:
OK, comments added.